internal/config/options_test.go

```diff
@@ -194,10 +194,10 @@ func Test_parseURLs(t *testing.T) {
 		if (err != nil) != test.wantErr {
 			t.Errorf("Failed to parse URLs %v: %s", test, err)
 		}
-		if o.AuthenticateURL != nil && o.AuthenticateURL.String() != test.authenticateURL {
+		if o.AuthenticateURL != (url.URL{}) && o.AuthenticateURL.String() != test.authenticateURL {
 			t.Errorf("Failed to update AuthenticateURL: %v", test)
 		}
-		if o.AuthorizeURL != nil && o.AuthorizeURL.String() != test.authorizeURL {
+		if o.AuthorizeURL != (url.URL{}) && o.AuthorizeURL.String() != test.authorizeURL {
```

Comment:
I converted these straight to default values from nil but I think just get rid of nil check?

Reply:
Yeah, I think it can go. Also, I love that the go playground now supports outside imports!
https://play.golang.org/p/3DW10L34lGJ

Reply:
Nice.  I saw that today but wasn't sure how new it was.